pkgs/misc/vim-plugins/overrides.nix

```
@@ -213,6 +213,30 @@ self: super: {
     meta.maintainers = with stdenv.lib.maintainers; [ minijackson ];
   };
 
+  LeaderF = super.LeaderF.overrideAttrs(old: {
+    buildInputs = [
+      python3
+      stdenv
```

stdenv shouldn't be necessary
```suggestion
      stdenv
```

---

@jonringer, is there a different way to get gcc? The plugin compiles a C extension.

---

oh hmmm. I think for the vimplugins they do a `{ stdenv = stdenvNoCC }`.... I'm not actually sure

---

I mean, it compiled fine for me when I used stdenv 🤷‍♂️

---

Actually, I just tried it without stdenv and it works fine. I'll remove the dependency. Thanks, @jonringer!